Cargo.toml

```
@@ -18,4 +18,7 @@ bindgen = "0.56"
 vcpkg = "0.2.8"
 
 [target.'cfg(target_os="macos")'.build-dependencies]
+pkg-config = "0.3.19"
+
+[target.'cfg(target_os="linux")'.build-dependencies]
```

I think this might work- I'll try it later
```suggestion
[target.'cfg(any(target_os="macos", target_os="linux"))'.build-dependencies]
pkg-config = "0.3.19"
```